File: atividade-3/011_min3/011_min3.js
//Pegar o menor número entre a, b e c

const { a, b, c } = { a: 1, b: 2, c: 3 };

const min3 = (a, b, c) => {
  if (a <= b && a <= c) return a;
  if (a > b && b < c) return b;

  return c;
};

console.log(min3(a, b, c));

File: atividade-3/modulo-3/011_min3/011_min3.hs
module Main where

--Pegar o menor número entre a, b e c

min3 :: Int -> Int -> Int -> Int
min3 a b c
   | a <= b && a <= c = a
   | a > b && b < c = b
   | otherwise = c

main = do
  a <- readLn :: IO Int
  b <- readLn :: IO Int
  c <- readLn :: IO Int
  print $ min3 a b c